site/content/3.12/data-science/graphml/_index.md

```diff
@@ -13,12 +13,60 @@ specialized teams of data scientists. ArangoGraphML, on the other hand,
 simplifies the utilization of GraphML, enabling a broader range of personas to
```

Comment:
> ArangoGraphML, on the other hand, simplifies the utilization of GraphML

Can we rename `ArangoGraphML` to just `ArangoML` here?

Reply:
Certainly, should be `ArangoML` here.